docs/getting-started/install/azure.md

```diff
@@ -0,0 +1,17 @@
+# Microsoft Azure
+
+- Generate an SSH key pair by executing:
+
+```
+ssh-keygen
+<enter a file name>
+<enter a password or no password for the key>
+```
+
+- Go to the [dokku on azure deployment page](https://github.com/Azure/azure-quickstart-templates/tree/master/dokku-vm) and click **Deploy to Azure**.
```

Comment:
This is maintained by the Azure team and the community. Anyone can submit a template to azure/azure-quickstart-templates. The dokku mantainers can easily send a PR as well. It uses your main installation script that uses apt-get. So any cahnges from there will just work as long as you don't change the script location.